### DISCUSSION TITLE
GMAIL Login issue

### DISCUSSION THREAD

**Discussion by Rajnparmar** [saran-surya/email_auth_node#35] [Q&A]
I/flutter (10819): email-auth >> Initialising Email-Auth server
I/flutter (10819): email-auth >> The remote server configurations are valid

Failed to send OTP ❌
I/flutter (10445): email-auth >> Message from server :: (535, b'5.7.8 Username and Password not accepted. Learn more at\n5.7.8  https://support.google.com/mail/?p=BadCredentials ci6-20020a05622a260600b003051ce7812asm4131738qtb.5 - gsmtp')
I/flutter (10445): Server error cannot send mail : section 1

I have set up the email_auth step by step but getting this error, and also the google has removed the setting to access for less secure app.., how can I allow it?

**Comment by saran-surya** [comment]
Are you creating a business account in GMAIL?
Do you have 2 step verification enabled ?

**Reply by vedantjk** [reply]
![image](https://user-images.githubusercontent.com/69514666/175337411-b8580601-0ba0-45fb-a8e8-d8824e671916.png)

Google has removed it from May 30, 2022

**Reply by Rajnparmar** [reply]
I/flutter ( 5405): email-auth >> Initialising Email-Auth server
I/flutter ( 5405): email-auth >> The remote server configurations are valid


I/flutter ( 5405): email-auth >> Failed to send OTP ❌
I/flutter ( 5405): email-auth >> Message from server :: (535, b'5.7.8 Username and Password not accepted. Learn more at\n5.7.8  https://support.google.com/mail/?p=BadCredentials b19-20020ae9eb13000000b006aee672937esm8028552qkg.37 - gsmtp')
I/flutter ( 5405): Server error cannot send mail : section 1

Still getting the same error used another new mail without any backup email etc, did all the steps again , should i use different mail than gmail?

**Comment by saran-surya** [comment]
This is a known issue, I am currently looking into an alternative solution Will be updating soon, Thankyou.

**Comment by saran-surya** [comment]
### Kindly use test server as of now.
Remote server patches are on the way, expect in 7 days

This should give you a good lead.
https://github.com/saran-surya/email_auth#update--google-turning-off-less-secure-apps--switching-to-oauth
